spec/takeaway_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'takeaway'
+describe Takeaway do
+
+  subject(:takeaway){described_class.new(:burger)}
+
+  describe '#show_menu'do
+    it 'shows the customer the available dishes and the prices'do
+    expect(takeaway.show_menu).to eq MenuList::MENULIST[:burger]
+    end
+  end
+
+  describe '#complete_order'do
+    it 'allows you to order'do
+    expect(takeaway.complete_order([[:Hamburger, :Chips],[1,2],[6.50]])).to eq true
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)